### PR TITLE
Ignore xcopy error for xml files to resolve CI issue

### PR DESCRIPTION
**Pull request by MaxWang-MS**
## Overview
Currently due to the way CI is set up the XML files are not produced causing the xcopy command to fail. This PR fixes the issue by suppressing the copy error on XML files.

### CODE REVIEW

**Comment by keveleigh** [comment]
>Currently due to the way CI is set up the XML files are not produced 

Should we update the CI to generate the XML files?

**Review by keveleigh — APPROVED** [review]

File: Assets/MRTK/Tools/MSBuild/MSBuildMRTKTemplates/SDKProjectTemplate.csproj.template [hunk 1]
```diff
@@ -150,7 +150,7 @@
   </Target>
 
   <Target Name="PostBuild" AfterTargets="PostBuildEvent">
-    <Exec Command="xcopy /y $(TargetDir)$(TargetName).dll $(PublishOutputPath)$(UnityConfiguration)\$(UnityPlatform)\&#xD;&#xA;xcopy /y $(TargetDir)$(TargetName).pdb $(PublishOutputPath)$(UnityConfiguration)\$(UnityPlatform)\&#xD;&#xA;xcopy /y $(TargetDir)$(TargetName).xml $(PublishOutputPath)$(UnityConfiguration)\$(UnityPlatform)\" />
+    <Exec Command="xcopy /y $(TargetDir)$(TargetName).dll $(PublishOutputPath)$(UnityConfiguration)\$(UnityPlatform)\&#xD;&#xA;xcopy /y $(TargetDir)$(TargetName).pdb $(PublishOutputPath)$(UnityConfiguration)\$(UnityPlatform)\&#xD;&#xA;xcopy /y $(TargetDir)$(TargetName).xml $(PublishOutputPath)$(UnityConfiguration)\$(UnityPlatform)\ || set errorlevel=0" />
```
Comment (keveleigh) [inline Assets/MRTK/Tools/MSBuild/MSBuildMRTKTemplates/SDKProjectTemplate.csproj.template:153]: Does this set the error level just for the last command, or for all of them?

Comment (MaxWang-MS) [reply Assets/MRTK/Tools/MSBuild/MSBuildMRTKTemplates/SDKProjectTemplate.csproj.template:153]: Just did an experiment locally and it seems that the error level applies to all three xcopy commands, but I do not expect it to be a problem as xcopy most likely fails only due to file not found issue here, and if that were to be the case there will be error stopping the process prior to the post build event.

**Pull request merged**